SysBot.Pokemon.YouTube/Helpers/YouTubeQueue.cs

```diff
@@ -0,0 +1,19 @@
+﻿using PKHeX.Core;
+
+namespace SysBot.Pokemon
+{
+    public class YouTubeQueue
+    {
```

Comment:
There's a lot of duplicate classes. Maybe these need to get pulled into the parent SysBot.Pokemon project as a reusable object?

Reply:
it depends, YouTube doesn't really have Commands nor whisper. if u ask me i would personally use it for Countdown only since we hit our Qouta Limit really fast per day.